api/pb/dart/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: threads_client_grpc
```

Comment:
why not match the js pattern (`threads-client-pb`)?

Reply:
oop, i thought that was the pattern

Reply:
yeah, i just copied that pattern from the js npm file https://www.npmjs.com/package/@textile/threads-client-grpc

Reply:
looks like both exist on npm though...

Reply:
oh right! forgot i changed it doh: https://github.com/textileio/go-threads/blob/master/api/pb/package.json#L2